doc/federation/federation.rst

```diff
@@ -0,0 +1,87 @@
+.. _federation/federation:
+
+========================
+The BinderHub Federation
+========================
+
+While it may seem like ``mybinder.org`` is a single website, it is
+in fact a **federation** of teams that deploy public BinderHubs to serve
+the community. This page lists the BinderHubs that currently help
+power ``mybinder.org``.
+
+Visiting ``mybinder.org`` will randomly redirect you to one
+of the following BinderHubs. You may also directly access a BinderHub
```

Comment:
I'd remove the sentence encouraging people to directly link to a particular cluster. I treat the fact that you can as an implementation detail people should not rely on but can use if they know what they are doing. A bit like Python classes let me poke around the insides of instances if I want to but it is clear that I am looking at the internals and that things might change without notice.